test/Mvc.Analyzers.Test/TestFiles/ApiConventionAnalyzerIntegrationTest/DiagnosticsAreReturned_IfMethodWithApiConventionMethod_ReturnsUndocumentedStatusCode.cs

```diff
@@ -0,0 +1,16 @@
+﻿using Microsoft.AspNetCore.Mvc;
+
+[assembly: ApiConventionType(typeof(DefaultApiConventions))]
+
+namespace Microsoft.AspNetCore.Mvc.Analyzers
+{
+    [ApiController]
+    public class DiagnosticsAreReturned_IfMethodWithApiConventionMethod_ReturnsUndocumentedStatusCode : ControllerBase
+    {
+        [ApiConventionMethod(typeof(DefaultApiConventions), nameof(DefaultApiConventions.Post))]
+        public IActionResult Get(int id)
+        {
+            /*MM*/return Accepted();
```

Comment:
Hmm?

Reply:
That's a marker for the analyzer - https://github.com/aspnet/Common/blob/master/src/Microsoft.AspNetCore.Analyzer.Testing/TestSource.cs#L12

Reply:
Ah ok.